Java/Stack/Stack.java

```diff
@@ -0,0 +1,77 @@
+public class Stack {
```

Comment:
@Khushi2402 

As we've added the stack program in PR #19 

So, this program is same as PR #19 or it's different?

I mean if both are the same, then we can remove this one because #19 is already merged.

Reply:
No, this is the Queue Program In Java

Reply:
@Khushi2402 
It will be good, if you review it again, because it seems that it's the `Stack` implementation.

Reply:
Okay sorry, I will add the Queue program again, you can remove this one.